crmd/control.c

```diff
@@ -1061,6 +1062,18 @@ config_query_callback(xmlNode * msg, int call_id, int rc, xmlNode * output, void
         fsa_cluster_name = strdup(value);
     }
 
+    {
+        sub_call_id = fsa_cib_conn->cmds->query(fsa_cib_conn, 
+            "/" XML_TAG_CIB "/" XML_CIB_TAG_CONFIGURATION
+            "/" XML_CIB_TAG_NOTIFICATIONS "/" XML_CIB_TAG_NOTIFY, NULL,
+            cib_scope_local | cib_xpath);
+
+        fsa_register_cib_callback(sub_call_id, FALSE, NULL,
+                                  notifications_query_callback);
+
+        crm_trace("Querying the CIB for notifications ... call %d", sub_call_id);
+    }
```

Comment:
Didn't want to change too much in the first attempt to create something working to be provided to pcs-development. Also wasn't sure if it would turn out to separate the handling
of that section completely from crm_config.
I also thought that changes wouldn't happen in these parts of the cib that
it would matter performance-wise. And it would complicate the parsing in the next step - right?
Anyway playing with xml-handling is good practice ;-) But please let me tackle that issue
after addressing the things that affect user behaviour.